### PR TITLE
Merge OGC 1.0, SMS (Tech) 2.0 and Data (Tech) 2.0 into master branch

### PR DESCRIPTION
**Pull request by disgruntled-patzer**
This PR concludes the summer development cycle of 2020 for the following packages/workspaces:

* Ops Ground Control 1.0
* SMS (Tech) 2.0
* Data (Tech) 2.0

More details in the Wiki and Changelogs

### CODE REVIEW

**Review by danipurwadi — APPROVED** [review]

File: ogc_ws/src/despatcher/scripts/g2a.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python3
+
```
Comment (rumesh986) [inline ogc_ws/src/despatcher/scripts/g2a.py:2]: Needs the GPL notice

File: ogc_ws/src/despatcher/scripts/missionserver.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
```
Comment (rumesh986) [inline ogc_ws/src/despatcher/scripts/missionserver.py:2]: Needs the GPL notice

File: ogc_ws/src/despatcher/src/update_mission_files.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+import time
+import rospy
+from pathlib import Path
+from despatcher.msg import LinkMessage
+from os import listdir
+
```
Comment (rumesh986) [inline ogc_ws/src/despatcher/src/update_mission_files.py:8]: Needs the GPL notice

File: ogc_ws/src/rff/src/rff.py [hunk 4]
```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+
```
Comment (rumesh986) [inline ogc_ws/src/rff/src/rff.py:2]: Needs the GPL notice

File: ogc_ws/src/rff/src/rff_old.py [hunk 5]
```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
```
Comment (rumesh986) [inline ogc_ws/src/rff/src/rff_old.py]: Are we keeping this file? it seems to be an older version of rff.py

**Review by rumesh986 — APPROVED** [review]
Some files are missing their GPL notice. These files are in rff, statustext, sbd/test. the rqt_mypkg file was also missing the notice. Otherwise it can be merged

**Comment by disgruntled-patzer** [comment]
GPL notices added. @huachen24 do we remove `rff_old.py`?

**Comment by huachen24** [comment]
> GPL notices added. @huachen24 do we remove `rff_old.py`?

Yup, we are no longer using this version anymore

**Review by huachen24 — APPROVED** [review]

**Pull request merged**